tutorials/python-for-loops/index.mdx

```diff
@@ -0,0 +1,933 @@
+
+The syntax for a for loop is as follows:
+
+```python
+for element in iterable-object:
+  statement(s)
+```
+
+Here are some basic examples of loops iterating over different types of objects:
+
+    <Tabs id="loop-examples">
+
+      <TabsTab label="List">
+        ```python
+        my_list = ["red", "blue", "yellow"]
+          
+        for color in my_list:
+          print(color)
+          
+        ## output:
+        red
+        blue
+        yellow
+        ```
+      </TabsTab>
+
+      <TabsTab label="String">
+        ```python
+        word = "hello"
+          
+        for letter in word:
+          print(letter)
+          
+        ## output:
+        h
+        e
+        l
+        l 
+        o
+        ```
+      </TabsTab>
+
+      <TabsTab label="Dictionary">
+        ```python
+        product_dict = {"bread": 1.5, "jam": 2.75, "butter": 0.99}
+          
+        for product in product_dict:
+          print(product)
+          
+        ### output:
+        bread
+        jam
+        butter
+          
+        for product in product_dict:
+          print(product_dict.get(product))
+          
+        ### output:
+        1.5
+        2.75
+        0.99
+        ```
+      </TabsTab>
+      <TabsTab label="Range">
+        ```python
+        for x in range (1, 4):
+          print(x)
+         
+        ## output:
+        1
+        2
+        3
+        ```
+      </TabsTab>
+    </Tabs>
+
+Ingredients of a for loop (see the syntax above):
+
+- **for**: this keyword is required to start a loop
+- **element**: after the `for` keyword, we can use any variable name we want! This variable will simply access each item of the sequence on each iteration. In the examples above you can see that we've called it variously `element`, `color`, `letter`, `product`, and `x`. Anything works, but generally you should try to give this variable a meaningful name: it will make your code easier to read for yourself and anyone else who looks at it.
+- **in**: this keyword is required to introduce the object you want to iterate over in the loop
+- **iterable-object**: after the `in` keyword, state the object you want to iterate over. This can be any iterable object, including lists, strings, dictionaries, ranges, tuples and sets. 
+- The for statement must end with a **colon**, followed by a **line return** and **indentation**.
+- **statement(s)**: One or more actions to be carried out on every iteration of the loop. In the examples above we simply use a print statement, but anything is possible.
+
+### Basic exercises
+
+    <Message type="tip">
+    When working with loops, you will probably find it more practical to use a [text editor and the terminal](/tutorials/get-started-python/#6:-from-the-shell-to-a-script) or [Jupyter notebook](https://jupyter.org/)/a , rather than the [Python Interactive Shell](/tutorials/get-started-python/#2:-getting-ready-to-code:-opening-the-python-interactive-shell).
```

Comment:
```suggestion
    When working with loops, you will probably find it more practical to use a [text editor and the terminal](/tutorials/get-started-python/#6:-from-the-shell-to-a-script) or [Jupyter notebook](https://jupyter.org/)/, rather than the [Python Interactive Shell](/tutorials/get-started-python/#2:-getting-ready-to-code:-opening-the-python-interactive-shell).
```